src/components/ReviewSummary/TokenReviewSummary.js

```diff
@@ -53,15 +62,26 @@ const TokenImage = styled(Image)`
 `;
 
 export const TokenReviewSummaryComponent = ({
-  assetSymbol, amount, rates, baseFiatCurrency, text, supportedAssets, theme, assetIcon, fiatAmount,
+  assetSymbol,
+  amount,
+  text,
+  assetIcon,
+  fiatAmount,
+  chain,
 }: Props) => {
+  const theme = useTheme();
+  const rates = useRates();
+  const baseFiatCurrency = useBaseFiatCurrency();
```

Comment:
This one defaults to `defaultFiatCurrency`
```suggestion
  const baseFiatCurrency = useFiatCurrency(); 
```